package-lock.json

```diff
@@ -297,13 +297,13 @@
 			}
```

Comment:
@jlawrence124 can you please unstage this file? As the goal of the PR is to include new functionality, not to upgrade dependencies. I will work on a PR to upgrade those but I'll need to make sure that there are no side effects

Reply:
done https://github.com/victorgz/vscode-salesforce-colorg/pull/4/commits/efe3b9045d2f8734a3da6412edf09fccc3575cfa